### PR TITLE
Add Oliveira-2014-517

### PR DESCRIPTION
**Pull request by Tarotis**
# Pull request checklist

- [x] add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

Here comes the mapped list from Oliveira. A short note on some of the columns, as I was not quite sure if any column would need some renaming:

- GLOSS contains the info from the proto-meaning; this information is sometimes given in Portuguese, and sometimes in Spanish, hence the neutral GLOSS. Could be renamed to "Concept_in_Source"?
- PROTO-ID is the ID given in the original source
- REFLEX_GLOSSES is the column that provides info about concepts from the reflexes that are reconstructed as part of the proto-language. In general, those are not directly mapped to concepticon, but do represent concepticon entries
- Given the mixed original glosses, ENGLISH is a custom translation, so it would be good if this could be checked.

Further, here comes the source: https://share.eva.mpg.de/index.php/s/HJDjMBQoKgNHMkf

### CODE REVIEW

**Comment by LinguList** [comment]
@JuarezRC, I hope you have received the invitation to Concepticon. Here, we discuss the new concept list, which was added by @tarotis (Frederic). @MuffinLinwist (Carlos, who's in the office) can help you in showing you how to browse the files. I'd like to ask you to check the English translations in this list, as well as check a bit if the mapping to concepticon makes sense. Here, we will introduce you to the new mapping principles over the next weeks.

**Comment by LinguList** [comment]
To compare, we recommend to check the online Concepticon website at https://concepticon.clld.org/

File: concepticondata/conceptlists/Oliveira-2014-517.tsv [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+Oliveira-2014-517-174	174	cará	yam		410	YAM		§171
+Oliveira-2014-517-175	175	lenha	firewood		10	FIREWOOD		§172
+Oliveira-2014-517-176	176	querer	to want		1784	WANT		§173
+Oliveira-2014-517-177	177	morcego	bat		1793	BAT		§174
+Oliveira-2014-517-178	178	costas (prefixo parte do corpo)	back (body-part prefix)					§175
+Oliveira-2014-517-179	179	costurar	to sew		1457	SEW		§176
+Oliveira-2014-517-180	180	vasilha, prato	bowl, plate					§177
+Oliveira-2014-517-181	181	duro, forte	hard, strong		1884	HARD		§178
+Oliveira-2014-517-182	182	pedaço, caco	piece, shard		594	PIECE		§179
+Oliveira-2014-517-183	183	grosso, espesso	think		1244	THICK		§180
```
Comment (JuarezRC) [inline concepticondata/conceptlists/Oliveira-2014-517.tsv:184]: There is a typo here; it should be thick

File: concepticondata/conceptlists/Oliveira-2014-517.tsv [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+Oliveira-2014-517-223	223	camarão	shrimp		2349	SHRIMP		§219
+Oliveira-2014-517-224	224	cabeça	head		1256	HEAD		§220
+Oliveira-2014-517-225	225	barro, poeira	clay, dust				CLAY//DUST	§221
+Oliveira-2014-517-226	226	esp. de planta	species of plant					§222
+Oliveira-2014-517-227	227	cutia (esp. de roedor)	cutia (species of rodent)		313	AGOUTI		§223
+Oliveira-2014-517-228	228	esp. de cabaça	species of gourd					§224
+Oliveira-2014-517-229	229	areia	sand		671	SAND		§225
+Oliveira-2014-517-230	230	pedra	stone		857	STONE		§226
+Oliveira-2014-517-231	231	urucum (a árvore e o seu fruto)	achiote		349	ACHIOTE		§227
+Oliveira-2014-517-232	232	encima, cume	atop		57	ATOP		§228
```
Comment (JuarezRC) [inline concepticondata/conceptlists/Oliveira-2014-517.tsv:233]: Include also the noun translation; it should be peak

File: concepticondata/conceptlists/Oliveira-2014-517.tsv [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
+Oliveira-2014-517-233	233	cortar o cabelo	to cut hair		2155	CUT (WITH SCISSORS)		§229
+Oliveira-2014-517-234	234	chifre de animal	animal horn		1393	HORN (ANATOMY)		§230
+Oliveira-2014-517-235	235	golpear a cabeça	to hit oneself's head					§231
```
Comment (JuarezRC) [inline concepticondata/conceptlists/Oliveira-2014-517.tsv:236]: Where is the reflexive meaning coming from?

File: concepticondata/conceptlists/Oliveira-2014-517.tsv [hunk 4]
```diff
@@ -0,0 +1,518 @@
...
+Oliveira-2014-517-260	260	soñar	dream		2374	DREAM		§256
+Oliveira-2014-517-261	261	abaixo, embaixo, sob	below, under		1485	BELOW OR UNDER		§257
+Oliveira-2014-517-262	262	carne	meat		634	MEAT		§258
+Oliveira-2014-517-263	263	jenipapo	genipa		3920	GENIPA		§259
+Oliveira-2014-517-264	264	colocar dentro, submergir	to put inside, to dive				INSERT//DIVE	§260
+Oliveira-2014-517-265	265	dentro, no centro	inside, in the center		1606	INSIDE		§261
+Oliveira-2014-517-266	266	banhar-se, tomar banho	to bathe, to take a bath		138	BATHE		§262
+Oliveira-2014-517-267	267	largo	wide		1243	WIDE		§263
+Oliveira-2014-517-268	268	morder	to bite		1403	BITE		§264
+Oliveira-2014-517-269	269	estrangeiro	stranger		791	STRANGER		§265
```
Comment (JuarezRC) [inline concepticondata/conceptlists/Oliveira-2014-517.tsv:270]: I think that 'foreign' works better

File: concepticondata/conceptlists/Oliveira-2014-517.tsv [hunk 5]
```diff
@@ -0,0 +1,518 @@
...
+Oliveira-2014-517-270	270	arco íris	rainbow		1733	RAINBOW		§266
+Oliveira-2014-517-271	271	capoeira, roça abandonada	abandoned rock					§267
```
Comment (JuarezRC) [inline concepticondata/conceptlists/Oliveira-2014-517.tsv:272]: 'abandoned field'; the Portuguese meanings talk more about a place/piece of land showing traces of past use

File: concepticondata/conceptlists/Oliveira-2014-517.tsv [hunk 6]
```diff
@@ -0,0 +1,518 @@
...
+Oliveira-2014-517-442	442	amendoim	groundnut		595	GROUNDNUT (PEANUT)		§439
+Oliveira-2014-517-443	443	bochecha	cheek		1730	CHEEK		§440
+Oliveira-2014-517-444	444	bochecha	cheek		1730	CHEEK		§441
+Oliveira-2014-517-445	445	esp. de palmeira	species of palm					§442
+Oliveira-2014-517-446	446	ponte, plataforma	bridge, platform		1011	BRIDGE		§443
+Oliveira-2014-517-447	447	raiz	root		670	ROOT		§444
+Oliveira-2014-517-448	448	pau velho, pau podre	old stick, rotten stick					§445
+Oliveira-2014-517-449	449	rodar	to run		1519	RUN		§446
+Oliveira-2014-517-450	450	roupa	clothes		1895	CLOTHES		§447
+Oliveira-2014-517-451	451	tipo de cesto	type of cedar					§448
```
Comment (JuarezRC) [inline concepticondata/conceptlists/Oliveira-2014-517.tsv:452]: 'type of basket'; I  guess that this word talks about a type of handicraft/home-made basket

File: concepticondata/conceptlists/Oliveira-2014-517.tsv [hunk 7]
```diff
@@ -0,0 +1,518 @@
...
+Oliveira-2014-517-480	480	sufixo verbal imperativo	verbal imperative suffix					§477
+Oliveira-2014-517-481	481	igarapé (?)	igarapé					§478
+Oliveira-2014-517-482	482	outro	other		197	OTHER		§479
+Oliveira-2014-517-483	483	um (numeral)	one		1493	ONE		§480
+Oliveira-2014-517-484	484	canela, parte inferior da perna	cinnamon, lower part of leg				LOWER LEG	§481
+Oliveira-2014-517-485	485	cheiro	smell		3704	ODOR (SMELL)		§482
+Oliveira-2014-517-486	486	remar	to row		1849	ROW		§483a
+Oliveira-2014-517-487	487	remo	oar		3137	OAR		§483b
+Oliveira-2014-517-488	488	remo	oar		3137	OAR		§484
+Oliveira-2014-517-489	489	borduna, bastão	stick		1295	STICK		§485
```
Comment (JuarezRC) [inline concepticondata/conceptlists/Oliveira-2014-517.tsv:490]: Borduna is more like a piece of wood used as a weapon, bastão stick should be 'walking stick'. I don't think that stick captures both meanings accurately.

File: concepticondata/conceptlists/Oliveira-2014-517.tsv [hunk 8]
```diff
@@ -0,0 +1,518 @@
...
+Oliveira-2014-517-505	505	animal	animal		619	ANIMAL		§501
+Oliveira-2014-517-506	506	calor, quente	heat, hot				HEAT//HOT	§502
+Oliveira-2014-517-507	507	ter febre, esquentar-se	to have fever, to heat up (reflexive)				HAVE A FEVER	§502a
+Oliveira-2014-517-508	508	feiticeiro	sorcerer		823	SORCERER		§503
+Oliveira-2014-517-509	509	esp. de pimenta	species of pepper					§504
+Oliveira-2014-517-510	510	perguntar	to ask		1309	ASK (INQUIRE)		§505
+Oliveira-2014-517-511	511	esp. de peixe	species of fish					§506
+Oliveira-2014-517-512	512	fio, linha, corda	thread, line, rope				THREAD//LINE//ROPE	§507
+Oliveira-2014-517-513	513	roubar	to steal		713	STEAL		§508
+Oliveira-2014-517-514	514	gente, corpo humano	person, human body		683	PERSON		§509
```
Comment (JuarezRC) [inline concepticondata/conceptlists/Oliveira-2014-517.tsv:515]: Should we use 'people' instead of person? It fits better to gente

**Review by JuarezRC — COMMENTED** [review]
I added some comments to individual entries. Let me know what you think. Also, this list includes meanings that are repeated twice; here are some examples: 9-10, 73-74, 84-85, 103-103-104, 284-285, 295-297, 381-382, 407-408, 412-413, 435-436, 470-471, 488-489. As far as I understand, these should be part of the same line, indicating that two different forms/words are listed with similar meanings?

File: concepticondata/conceptlists/Oliveira-2014-517.tsv (hunk 2 above)
Comment (Tarotis) [reply concepticondata/conceptlists/Oliveira-2014-517.tsv:233]: Following this, I unmapped the entry

File: concepticondata/conceptlists/Oliveira-2014-517.tsv (hunk 3 above)
Comment (Tarotis) [reply concepticondata/conceptlists/Oliveira-2014-517.tsv:236]: I couldn't figure out a better translation. Do you have one? For me, the Portuguese entry also has a primarily reflexive reading, as you wouldn't hit the head of somebody else but rather your own.

Comment (JuarezRC) [reply concepticondata/conceptlists/Oliveira-2014-517.tsv:236]: Ok, then I think it could be something like golpear-se a cabeça in Portuguese.

Comment (Tarotis) [reply concepticondata/conceptlists/Oliveira-2014-517.tsv:236]: If you say that the form as it is (without reflexive) does not have that reflexive reading, we should not change the entry but rather the English translation. But, again, I wouldn't know which better translation to chose.

**Comment by Tarotis** [comment]
I changed everything according to Cristian's comments. @LinguList Should there be a second review?

**Comment by LinguList** [comment]
Yes, by myself, but I cannot promise to be able to finish today.

**Comment by Tarotis** [comment]
There were some problems in the data with missing proto-glosses. In those cases, the anterior gloss was applied automatically, which is of course wrong. I fixed this by majority-principle, except for one case, in which the concept "unclear" was chosen ("?" threw an error). There should be no further major changes now.

**Review by LinguList — APPROVED** [review]

**Pull request merged**